site/app/templates/admin/Configuration.twig

```diff
@@ -224,7 +224,9 @@
                         </option>
                     {% endfor %}
                 </select>
-                 <a href="{{ email_room_seating_url }}" class="btn btn-primary" id="email_seating_assignment" style="margin-top: 25px">Email Seating Assignments</a>
+                <br>
```

Comment:
We probably want this a tag somewhere (probably under the new instructor only feature) to actually send the emails?